Media/FFmpegMovie/FFmpegMovieFrameInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace OpenOrchestra\Media\FFmpegMovie;
+
+/**
+ * Interface FFmpegMovieFrameInterface
+ */
+interface FFmpegMovieFrameInterface
```

Comment:
I suggest to rename this interface into something more genereic not mentionning FFmpeg, for instance VideoManagerInterface. That way FFmpegMovieFrame is one implementation among others. In the VideoToImageManager, you inject a VideoManagerInterface that can use ffmep or anything else dealing with videos.